src/appwrite.ts

```diff
@@ -17,7 +17,7 @@ export const createMagicLink = async (id: string, email: string) => {
       email,
       "http://localhost:5173/App"
```

Comment:
This should be an env variable, so that you can easily set it from Netlify as well